packages/etherlime-config/index.js

```diff
@@ -0,0 +1,4 @@
+const ganacheSetupConfig = require('./ganacheSetup.json');
+
+module.exports = ganacheSetupConfig
```

Comment:
In order not to break future versions, if we decide to add more things to this config, lets export this as `module.exports = { ganacheSetupConfig }`. Then we will include it as `const { ganacheSetupConfig } = require('etherlime-config');`